### PR TITLE
clean up issues

### PR DESCRIPTION
**Pull request by markbainix**
refactor .click() to COM.click_button()

### CODE REVIEW

File: keywords/webui/apps.py [hunk 1]
```diff
@@ -406,7 +407,8 @@ def navigate_to_app_section(cls, name: str) -> None:
         Example:
             - Apps.navigate_to_app_section('WG Easy')
         """
-        WebUI.xpath(xpaths.common_xpaths.any_xpath(f'//*[@class="section ng-star-inserted" and contains(text(),"{name}")]')).click()
+        obj = WebUI.xpath(xpaths.common_xpaths.any_xpath(f'//*[@class="section ng-star-inserted" and contains(text(),"{name}")]'))
+        obj.click()
```
Comment (ericbsd) [inline keywords/webui/apps.py:411]: You can use `Common.click_on_element(xpath)` for these.

File: keywords/webui/local_groups.py [hunk 2]
```diff
@@ -288,7 +288,8 @@ def delete_group_allowed_sudo_command(cls, command: str) -> None:
          - Local_Users.delete_group_allowed_sudo_command('/usr/sbin/zfs')
         """
         if COM.is_visible(xpaths.common_xpaths.any_pill_delete('sudo-commands', command)):
-            WebUI.xpath(xpaths.common_xpaths.any_pill_delete('sudo-commands', command)).click()
+            obj = WebUI.xpath(xpaths.common_xpaths.any_pill_delete('sudo-commands', command))
+            obj.click()
```
Comment (ericbsd) [inline keywords/webui/local_groups.py:292]: Same here, `Common.click_on_element(xpath)`.

File: keywords/webui/local_users.py [hunk 3]
```diff
@@ -484,7 +484,8 @@ def confirm_delete_user_by_full_name(cls, fullname: str, primary_group: bool = F
             cls.click_user_delete_button(cls.get_username_from_full_name(fullname))
             if primary_group:
                 if COM.is_visible(xpaths.common_xpaths.checkbox_field('delete-primary-group')):
-                    WebUI.xpath(xpaths.common_xpaths.checkbox_field('delete-primary-group')).click()
+                    obj = WebUI.xpath(xpaths.common_xpaths.checkbox_field('delete-primary-group'))
+                    obj.click()
```
Comment (ericbsd) [inline keywords/webui/local_users.py:488]: Same here, `Common.click_on_element(xpath)`.

File: keywords/webui/login_page_ui.py [hunk 4]
```diff
@@ -124,7 +124,8 @@ def assert_ixsystems_link_is_correct(cls) -> bool:
         :return: true if iXSystem link is correct
         """
         assert WebUI.current_url() == 'http://'+private_config["IP"]+'/ui/sessions/signin'
-        WebUI.xpath(xpaths.common_xpaths.any_xpath('//img[@src="assets/images/ix_logo_full.png"]')).click()
+        obj = WebUI.xpath(xpaths.common_xpaths.any_xpath('//img[@src="assets/images/ix_logo_full.png"]'))
+        obj.click()
```
Comment (ericbsd) [inline keywords/webui/login_page_ui.py:128]: Same here, `Common.click_on_element(xpath)`.

File: keywords/webui/replication.py [hunk 5]
```diff
@@ -55,10 +55,11 @@ def delete_replication_task_by_name(cls, name: str) -> None:
             - Replication.delete_replication_task_by_name('myRepTask')
         """
         if cls.is_replication_task_visible(name):
-            WebUI.xpath(xpaths.common_xpaths.any_child_parent_target(
+            obj = WebUI.xpath(xpaths.common_xpaths.any_child_parent_target(
                     f'//*[contains(text(),"{name}")]',
                     'tr',
-                    f'button[contains(@data-test,"-delete")]')).click()
+                    f'button[contains(@data-test,"-delete")]'))
+            obj.click()
```
Comment (ericbsd) [inline keywords/webui/replication.py:62]: Same here, `Common.click_on_element(xpath)`.

File: keywords/webui/smb.py [hunk 6]
```diff
@@ -44,15 +44,16 @@ def click_edit_share_filesystem_acl(cls, name: str) -> None:
...
         assert WebUI.wait_until_visible(xpaths.common_xpaths.any_header(f'Edit ACL', 1)) is True
 
     @classmethod
     def delete_share_by_name(cls, sharetype: str, name: str, action: str) -> None:
         """
         This method deletes the given share on the Shares page
         """
-        WebUI.xpath(xpaths.common_xpaths.button_share_action_by_name(sharetype, name, action)).click()
+        obj = WebUI.xpath(xpaths.common_xpaths.button_share_action_by_name(sharetype, name, action))
+        obj.click()
```
Comment (ericbsd) [inline keywords/webui/smb.py:56]: Same here, `Common.click_on_element(xpath)`.

File: keywords/webui/smb.py [hunk 7]
```diff
@@ -84,19 +85,23 @@ def set_ignore_list(cls, name: str) -> None:
         :param name: name of the account to ignore
         """
         assert COM.is_visible(xpaths.common_xpaths.input_field('ignore-list'))
-        WebUI.xpath(xpaths.common_xpaths.input_field('ignore-list')).click()
+        obj = WebUI.xpath(xpaths.common_xpaths.input_field('ignore-list'))
+        obj.click()
         name = COM.convert_to_tag_format('ignore-list-'+name)
-        WebUI.xpath(xpaths.common_xpaths.option_field(name)).click()
+        obj = WebUI.xpath(xpaths.common_xpaths.option_field(name))
+        obj.click()
```
Comment (ericbsd) [inline keywords/webui/smb.py:92]: Same here, `Common.click_on_element(xpath)`.

File: keywords/webui/smb.py [hunk 8]
```diff
@@ -115,6 +120,9 @@ def set_watch_list(cls, name: str) -> None:
         :param name: name of the account to watch
         """
         assert COM.is_visible(xpaths.common_xpaths.input_field('watch-list'))
-        WebUI.xpath(xpaths.common_xpaths.input_field('watch-list')).click()
+        obj = WebUI.xpath(xpaths.common_xpaths.input_field('watch-list'))
+        obj.click()
```
Comment (ericbsd) [inline keywords/webui/smb.py:124]: Same here, `Common.click_on_element(xpath)`.

**Review by ericbsd — CHANGES_REQUESTED** [review]

File: keywords/webui/apps.py (hunk 1 above)
Comment (markbainix) [reply keywords/webui/apps.py]: resolved

File: keywords/webui/local_groups.py (hunk 2 above)
Comment (markbainix) [reply keywords/webui/local_groups.py]: resolved

File: keywords/webui/local_users.py (hunk 3 above)
Comment (markbainix) [reply keywords/webui/local_users.py]: resolved

File: keywords/webui/login_page_ui.py (hunk 4 above)
Comment (markbainix) [reply keywords/webui/login_page_ui.py]: resolved

File: keywords/webui/smb.py (hunk 6 above)
Comment (markbainix) [reply keywords/webui/smb.py]: resolved

File: keywords/webui/smb.py (hunk 7 above)
Comment (markbainix) [reply keywords/webui/smb.py]: resolved

File: keywords/webui/smb.py (hunk 8 above)
Comment (markbainix) [reply keywords/webui/smb.py]: resolved

**Pull request merged**